doc/decisions/decision_process.md

```diff
@@ -80,6 +84,8 @@ The first step is to create a PR with:
 - **one** decision, where at least the "Problem" and "Considered Alternatives" are filled out.
 - a link from [README.md](README.md) from the "Drafts" section to this decision.
 
+> At least the problem must be clear to everyone involved before the decision can leave the "Drafts" status.
```

Comment:
I agree with that. But as you wrote 2 lines above: Just because the problem isn't clear to everyone, doesn't mean we can't add new "Considered Alternatives" already.

Reply:
Actually it is even required to have "Considered Alternatives". Any further clarifications needed?